docs/customer-360/aviation/process.sql

```diff
@@ -0,0 +1,86 @@
+SET 'auto.offset.reset' = 'earliest';
+
+CREATE TABLE customers (ID             INT     PRIMARY KEY
+                       , NAME           VARCHAR
+                       , ADDRESS        VARCHAR
+                       , EMAIL          VARCHAR
+                       , PHONE          VARCHAR
+                       , LOYALTY_STATUS VARCHAR)
+              WITH (KAFKA_TOPIC='customers'
+                   , FORMAT='AVRO'
+                   , PARTITIONS=6);
+
+CREATE TABLE flights (ID               INT     PRIMARY KEY
```

Comment:
as above

Reply:
Formatting concerns for this recipe are covered as part of https://github.com/confluentinc/ksqldb-recipes/issues/90 (cc: @rmoff )